uber/configspec.ini

```diff
@@ -316,6 +316,12 @@ blank_staff_badges = integer(default=50)
 # This setting determines how many days in advance we begin tracking setup shifts.
 setup_shift_days = integer(default=5)
 
+# Some events may wish to have JIRA integration enabled so that admins can
```

Comment:
Oops, this change shouldn't be in this PR. I'll roll it back, plus the corresponding changes to base.html.